### PR TITLE
[Usage.md] Added font matter

### PR DESCRIPTION
**Pull request by Confectrician**
Ongoing work for #33 
Reference: openhab/openhab-docs#690

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by ghys** [comment]
@Confectrician can you add:
```
description: ""Google Assistant is Google's virtual personal assistant and uses Actions on Google as the platform for \"Actions\" (software applications) to extend the functionality of the Google Assistant."
```
to the front matter?
It's transformed into a meta tag by the website, useful for Twitter cards etc.

File: USAGE.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+---
+layout: documentation
+title: Google Assistant
```
Comment (ThomDietrich) [inline USAGE.md]: "Google Assistant _what_"

Comment (Confectrician) [reply USAGE.md]: I just thought about really adding a `what` now, just for the sake of fun. 😄 

Is it called *skill* too there?
Or should we choose *integration*?

Comment (ThomDietrich) [reply USAGE.md]: 😄 sorry

Hmm it's called "Action" in the Google Assistant world. https://assistant.google.com/services/a/uid/000000f5c61c627e?hl=en-US
I would like integration better. @marziman wdyt?

**Comment by ThomDietrich** [comment]
One more thing regarding the image. The position below the `docs/` folder is a bit inconvenient. I'd suggest to restructure just as in the HABPanel project: https://github.com/openhab/org.openhab.ui.habpanel/tree/master/doc

* create a doc or docs folder
* move USAGE.md there
* move the image in the images folder below

This way the markdown file can contain an `images/....png` link

File: USAGE.md (hunk 1 above)
Comment (Confectrician) [reply USAGE.md]: +1 for integration

Lets wait for feedback.

**Comment by Confectrician** [comment]
Maybe something we should check for all oh repos where we grab external content and migrate to a generic solution.

**Comment by ThomDietrich** [comment]
Yes I'd like that. The whole "retrieve external content" script in docs developed over time and now would be a good time to push for a generic solution. I'd vote for `docs/`, `docs/images/` and `docs/USAGE.md`.

**Comment by Confectrician** [comment]
No need for long discussions. 😃 
I will provide PRs then.

**Comment by Confectrician** [comment]
Hey @marziman,

Just in time, i wanted to comment here today or tomorrow. :slightly_smiling_face:

My suggestion:
Merge this PR. (We can do further improvements later and the website will already improve with the additional front matter.)

Then change the structure as @ThomDietrich suggested.
I will provide the same structure for other repos, while continouing this topic.
Also i will then update and merge the corresponding openhab-docs pr to integrate this repo in the automatic website builds.


BR
Jerome

**Comment by marziman** [comment]
ok merging it...

**Pull request merged**